<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintBrushSettingsWidget</class>
 <widget class="QWidget" name="PaintBrushSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="buttonRoundCap">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons.qrc">
       <normaloff>:/brushes/assets/brushes/circle.png</normaloff>:/brushes/assets/brushes/circle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="buttonSquareCap">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons.qrc">
       <normaloff>:/brushes/assets/brushes/square.png</normaloff>:/brushes/assets/brushes/square.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelRadius">
     <property name="text">
      <string>&amp;Radius:</string>
     </property>
     <property name="buddy">
      <cstring>sliderRadius</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelRadiusValue">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QSlider" name="sliderRadius">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelPressureValue">
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QSlider" name="pressureSlider">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="cbAntialias">
     <property name="text">
      <string>&amp;Anti-aliasing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
